recipe/meta.yaml

```diff
@@ -11,13 +11,18 @@ source:
 
 build:
     number: 0
+    script:
+        - python setup.py build_ext --inplace --compiler=mingw32  # [win and py<35]
+        - python setup.py install --skip-build  # [win and py<35]
+        - python setup.py install --single-version-externally-managed --record record.txt  # [not (win and py<35)]
```

Comment:
`*nix` and Python 3.5 on Windows is fine, but Python 2.7 and 3.4 complains about a missing symbol:

```shell
Creating library build\temp.win32-2.7\Release\gridfill\_gridfill.lib and object build\temp.win32-2.7\Release\gridfill\_gridfill.exp
_gridfill.obj : error LNK2019: unresolved external symbol _fmax referenced in function ___pyx_f_8gridfill_9_gridfill_poisson_fill
build\lib.win32-2.7\gridfill\_gridfill.pyd : fatal error LNK1120: 1 unresolved externals
error: command 'C:\\Users\\appveyor\\AppData\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\link.exe' failed with exit status 1120
```

I can compile it locally using `mingwpy`, so I am trying to use workaround that problem... Fingers crossed.